content/docs/guides/crosswalk/kubernetes/aws/02-kubernetes-stacks/01-cluster-configuration.md

```diff
@@ -0,0 +1,672 @@
+---
+
+title: Kubernetes Cluster
+
+aliases: ["/docs/guides/crosswalk/kubernetes/aws/cluster/"]
+---
+
+Manage the control plane, worker nodes, RBAC, and defaults.
+
+  * [Create the Control Plane](#create-the-control-plane)
+  * [Create the Worker Nodes](#create-the-worker-nodes)
+  * [Use the Cluster](#use-the-cluster)
+  * [Setting Defaults](#setting-defaults)
+  * [Use Kuberntes RBAC](#use-kubernetes-rbac)
+  * [Updating the Worker Nodes](#updating-the-worker-nodes)
+
+
+The full code for this stack is on [GitHub][gh-repo-stack].
+
+See the [official Kubernetes docs][k8s-docs] for more details.
+
+## Create the Control Plane
+
+Create a new EKS cluster control plane equipped with:
```

Comment:
This reads a bit funny the way it's worded and maybe a bit out of order. Typically, we tell you

1. What you're going to do in a succinct, understandable way
2. How to do it (e.g., show the CLI command and/or code)
3. Give you the full details of what has been done

In that sense, leading with the bullets seems a bit detailed/daunting IMHO.

Reply:
Done